source/_components/remote.harmony.markdown

```diff
@@ -24,14 +24,28 @@ Supported units:
 - Harmony Elite
 
 
-To use your Harmony remote in your installation, add the following to your `configuration.yaml` file:
+The preferred way to setup the Harmony remote is by enabling the [discovery component](/components/discovery/).
+
+You can override some discovered values (e.g. the `port` or `activity`) by adding a `configuration.yaml` setting.
+In this case the `name` used in the config must match exactly the name you have chosen for your hub. You must also leave the `host`
+setting empty to force the platform to discover the host IP automatically.
+
+If you want to manually configure all device settings, you will need to add its settings to your `configuration.yaml`.
 
 ```yaml
-# Example configuration.yaml entry
+# Example configuration.yaml entry if you need any manual configuration
```

Comment:
Avoid changing the default example since it often copy and pasted.
Distill it down to it's base components and instead include and advanced example.